strings.js

```diff
@@ -0,0 +1,263 @@
+console.log(sentence + "\n" + withoutLast);
+
+// ## Bonus Questions
+//
+// 1. Write a code block that adds a "#" to a string if the string has an odd
+// number of characters, otherwise leave it as is. print/log the result
+//
+//     Examples:
+//
+//   ```js
+//   let name = 'taq'  // should log 'taq#'
+//   let name2 ='john' // should log 'john'
+//   ```
+
+let oddString = "phone";
+if (oddString.length % 2 === 1) {
+  console.log(oddString + "#");
+} else {
+  console.log(oddString);
+}
+
+
+// 2. Write a code block that prints/logs the 3rd to last characters of a string.
+// If the string is not at least 3 characters long then log an error message.
+// <details>
+//     <summary>
+//         Hint
+//     </summary>
+//     It should work for strings of any length.
+// </details>
+//
+// Examples, for variables:
+// ```js
+// let name1 = 'Taq' // Should log 'Taq'
+// let sentence = 'He is Taq'// Should log 'Taq'
+// ```
+
+let threat = "You need to back off before I eliminate you";
+if (threat.length < 3) {
+  console.log("#ERROR#");
+} else if (threat.length >= 3) {
+  console.log(threat.slice(threat.length - 3));
+}
+
+let threat2 = "Hi";
+if (threat2.length < 3) {
+  console.log("#ERROR#");
+} else if (threat2.length >= 3) {
+  console.log(threat2.slice(threat2.length - 3));
+}
+
+//
+// 3. Write a code block that prints/logs whether a string `x` starts with a
+// vowel.
+// <details>
+//   <summary>
+//     Extra
+//   </summary>
+//
+//   Make it [case insensitive](https://en.wikipedia.org/wiki/Case_sensitivity)
+//
+// </details>
+//
+
+let stringX = "all of you are fired!";
+if (stringX[0] === "A") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "E") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "I") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "O") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "U") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "a") {
+  console.log("This string starts with a lowercase vowel.");
+} else if (stringX[0] === "e") {
+  console.log("This string starts with a lowercase vowel.");
+} else if (stringX[0] === "i") {
+  console.log("This string starts with a lowercase vowel.");
+} else if (stringX[0] === "o") {
+  console.log("This string starts with a lowercase vowel.");
+} else if (stringX[0] === "u") {
+  console.log("This string starts with a lowercase vowel.");
+}
+
+//
+// 4. Write a code block that given two strings, `str1` and `str2`,
+// prints/logs a new string with `str2` mashed into the middle of `str1`
+//
+// Examples:
+//
+//   - For `str1` = 'help' and `str2` = 'me'. Your program should log `'hemelp'`
+//
+//    - For `str1` = 'hello' and `str2` = 'world'. Your program should log
+//    `'helworldlo'`
+//
+
+let str1 = "hello";
+let str2 = "world";
+console.log(str1.slice(0, 3) + str2 + str1.slice(3));
```

Comment:
Would it be possible to change (str1.slice(0, 3) to (str1.slice(0, str1.length / 2) ? Again, your code would give me wrong output if we change str1 to "Konnichiwa".